packages/office-ui-fabric-react/src/components/Callout/Callout.doc.tsx

```diff
@@ -25,26 +34,31 @@ export const CalloutPageProps: IDocPageProps = {
     {
       title: 'Default Callout',
       code: CalloutBasicExampleCode,
+      codepenJS: CalloutBasicExampleCodepen,
```

Comment:
These aren't needed now (and will actually cause compile errors if you update from master)

Reply:
Removed!